### ISSUE TITLE
Micrograph class is not needed

### ISSUE THREAD

**Issue by janden** [ComputationalCryoEM/ASPIRE-Python#204]
Right now, it does the same thing as `Image`, plus some preprocessing and filtering stuff (regarding margins) that can be moved elsewhere, possibly into `Image`.

**Comment by junchaoxia** [comment]
I believe this class was proposed for particle picking and CTF estimation modules. I will assign it to Partice Picking project temporarily.

**Comment by janden** [comment]
I think that's correct. Still, it doesn't do anything that's too specific for the `Image` class.

**Comment by garrettwrong** [comment]
I think we can drop these...

```
(base) ➜  ASPIRE-Python git:(fix_sphinx_lang) ✗ git grep "Micrograph(" 
src/aspire/source/mrcstack.py:        self.im = Micrograph(filepath, square=True).im
tests/test_CtfEstimate.py:    def testRectangularMicrograph(self, slice_range):
tests/test_micrograph.py:            micrograph = Micrograph(path, margin=100, shrink_factor=2)
tests/test_micrograph.py:            micrograph = Micrograph(path)
(base) ➜  ASPIRE-Python git:(fix_sphinx_lang) ✗ git grep MrcStack   
src/aspire/source/mrcstack.py:class MrcStack(ImageSource):
tests/test_mrc_stack.py:from aspire.source.mrcstack import MrcStack
tests/test_mrc_stack.py:            mrc_stack = MrcStack(path)
tests/test_mrc_stack.py:            mrc_stack = MrcStack(path)
```

**Comment by chris-langfield** [comment]
Closed by #718
